### PR TITLE
Add windows support

### PR DESCRIPTION
**Pull request by tadeu**

### CODE REVIEW

File: recipe/meta.yaml
Comment (tadeu) [inline recipe/meta.yaml]: @doxygen Are you ok being added as a co-maintainer of this conda-forge package? This will give you read/write/merge privileges on the resulting feedstock repository.

File: recipe/meta.yaml
Comment (isuruf) [inline recipe/meta.yaml:36]: I'd rather not depend on a dll like this as this might interfere with another package providing `libclang.dll`.
Is it hard to build on windows? If so, then I'm okay with repackaging the binary.

**Review by isuruf — COMMENTED** [review]
You need to rerender for a change like this.

File: recipe/meta.yaml
Comment (ocefpaf) [inline recipe/meta.yaml:21]: If you are removing the Windows `skip` you need to keep the `vc`s.
See http://conda-forge.github.io/docs/buildwin.html#vc-features

Comment (isuruf) [reply recipe/meta.yaml:21]: I don't think you need them. This is just repackaging binaries.

Comment (ocefpaf) [reply recipe/meta.yaml:21]: I missed that. In that case you are correct.

File: recipe/meta.yaml
Comment (tadeu) [reply recipe/meta.yaml:36]: I'll see if it's possible to build on Windows with `libclang` statically linked.

Comment (tadeu) [reply recipe/meta.yaml:36]: @isuruf It's easy to build it on Windows, but _without clang support_. There are no precompiled _static_ version of `libclang` on Windows.

Unfortunately, Doxygen depends on clang for "a more accurate source browsing experience, and better syntax highlighting, cross referencing, and call graphs" [(ref.)](http://lists.llvm.org/pipermail/cfe-dev/2013-May/029732.html).

I agree this could be a problem, but I think there are no other packages using `libclang.dll`, and there are no `libclang` packages for Windows. Whenever `libclang` or `clang` is added to conda-forge, we can update this recipe to depend on it. What do you think?

Comment (isuruf) [reply recipe/meta.yaml:36]: Sure, we can have it as it is until, https://github.com/conda-forge/staged-recipes/pull/1481 is merged.

**Comment by jakirkham** [comment]
Added PR ( https://github.com/tadeu/doxygen-feedstock/pull/1 ) against your fork to re-render this PR in response to the issues you had raised, @tadeu . Please take a look.

**Comment by tadeu** [comment]
Thanks @jakirkham. I had already fixed the problem manually (I rerendered removing the `skip` line, then put the skip line back again), so I think these changes won't be needed.

File: recipe/meta.yaml
Comment (tadeu) [reply recipe/meta.yaml]: @doxygen Since you didn't answer, I'm removing your name from the list in order to get this PR ready for merging, but feel free to join at any time :)

**Comment by tadeu** [comment]
@isuruf Can it be merged? Thanks!

**Review by isuruf — APPROVED** [review]

**Comment by isuruf** [comment]
I'll merge this once the tests pass.

File: recipe/LICENSE
Comment (jakirkham) [inline recipe/LICENSE:1]: Why do we need this license file?

**Comment by jakirkham** [comment]
> Thanks @jakirkham. I had already fixed the problem manually (I rerendered removing the `skip` line, then put the skip line back again), so I think these changes won't be needed.

So the problem with this is that the re-rendering bot or another user might make the same mistake. 

Also there is another change in my PR to make sure the team update script doesn't choke on this recipe when parsing through it for maintainers.

Could you please a similar change to your PR or merge mine? Either way is fine with me.

**Comment by tadeu** [comment]
@isuruf Could this be merged now? Thanks!

**Comment by isuruf** [comment]
Thanks for the PR, @tadeu.

**Pull request merged**
